scripts/dts/gen_driver_kconfig_dts.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 Kumar Gala <galak@kernel.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+import re
+from collections import defaultdict
+
+import yaml
+try:
+    # Use the C LibYAML parser if available, rather than the Python parser.
+    # This makes e.g. gen_defines.py more than twice as fast.
+    from yaml import CLoader as Loader
+except ImportError:
+    from yaml import Loader     # type: ignore
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'python-devicetree',
+                                'src'))
+
+def binding_paths(bindings_dirs):
+    # Returns a list with the paths to all bindings (.yaml files) in
+    # 'bindings_dirs'
+
+    binding_paths = []
+
+    for bindings_dir in bindings_dirs:
+        for root, _, filenames in os.walk(bindings_dir):
+            for filename in filenames:
+                if filename.endswith(".yaml") or filename.endswith(".yml"):
+                    binding_paths.append(os.path.join(root, filename))
```

Comment:
suggestion: pathlib glob could likely simplify this (I think)

Reply:
This is pretty much lifted from edtlib.py so going to leave it alone for now.